core/loader.js

```diff
@@ -218,7 +218,7 @@ if ( !CKEDITOR.loader ) {
 				} else {
 					// Append this script to the list of loaded scripts.
 					this.loadedScripts.push( scriptName );
-
+					
 					document.write( '<script src="' + scriptSrc + '" type="text/javascript"><\/script>' );
```

Comment:
Let's try making this CSP compatible in a separate PR (document.createElement)

Reply:
Added as comment in the CSP PR